Element84/cmr-stac-api-proxy
# Add static OpenAPI 3.0 documentation

Generated static documentation from the OpenAPI specification and deploy as part of the application. Serving this static file reduces the amount of time the Lambda executes. Having the files statically generate will need some updates to make sure that the server location is updated once deployed.

To test:
- `npm start` in the search directory
- ensure that navigating to `/` will redirect you to `/docs/index.html` and show the new documentation
- `npm run build:docs` to show that the new index.html file will be updated from the swagger config file.
